{
  "info": {
    "author": "Mohd. Shafikur Rahman",
    "author_email": "shafikshaon@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/shafikshaon/django-debug-helper",
    "keywords": "",
    "license": "",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-debug-helper",
    "package_url": "https://pypi.org/project/django-debug-helper/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-debug-helper/",
    "project_urls": {
      "Homepage": "https://github.com/shafikshaon/django-debug-helper"
    },
    "release_url": "https://pypi.org/project/django-debug-helper/1.1.0/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Django debug error related exception search on Google/ Stack Overflow.",
    "version": "1.1.0",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 4471849,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "a52eb99dffb32463bc068cdc89a76b22ff069ec1f5431e8620f28514346847a3",
        "md5": "d965c28ecf80eb387e66b2f8ff5a3e05",
        "sha256": "cc772f0872e36a7b1ae0cd3eb196930277602f312c3e5eab37f434f6c90f3f3c"
      },
      "downloads": -1,
      "filename": "django_debug_helper-1.1.0-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "d965c28ecf80eb387e66b2f8ff5a3e05",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": null,
      "size": 2127,
      "upload_time": "2018-11-09T19:00:31",
      "upload_time_iso_8601": "2018-11-09T19:00:31.886783Z",
      "url": "https://files.pythonhosted.org/packages/a5/2e/b99dffb32463bc068cdc89a76b22ff069ec1f5431e8620f28514346847a3/django_debug_helper-1.1.0-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "529208157501aeedd2dcfbc9ad381bcdfa31de20b700e902d9715ed175d819eb",
        "md5": "7646fdd21e449db25233de93ab03b1fd",
        "sha256": "ac2767d7b2fd61a41eefd69184aa47a5ce83cd2d6567cdeffa8fafcc09967dcd"
      },
      "downloads": -1,
      "filename": "django-debug-helper-1.1.0.tar.gz",
      "has_sig": false,
      "md5_digest": "7646fdd21e449db25233de93ab03b1fd",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 1246,
      "upload_time": "2018-11-09T19:00:33",
      "upload_time_iso_8601": "2018-11-09T19:00:33.438581Z",
      "url": "https://files.pythonhosted.org/packages/52/92/08157501aeedd2dcfbc9ad381bcdfa31de20b700e902d9715ed175d819eb/django-debug-helper-1.1.0.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}